lifetiles-graph/src/main/java/nl/tudelft/lifetiles/graph/view/TileView.java

```diff
@@ -292,3 +290,64 @@ private void segmentInsert(final int index, final SequenceSegment segment) {
     }
 
 }
+
+/**
+ * This class holds additional information to be drawn on the screen and also
+ * the lane index for the vertex.
+ */
+class Container {
```

Comment:
Renamed but, this class is needed because there are no other methods in java to store multiple variables.